File: JavaScript/LinearSearch.js
"use strict"

let arr = [1, 3, 2, 5, 6, 9, 0, 8, 7, 4];

let target = -1;
let found = -1;
for (let i = 0; i < arr.length; i++) {
    if (arr[i] == target) {
        found = i;
        break;
    }
}

console.log(arr.join());
if (found != -1)
    console.log(`${target} is in the index ${found}`, target, found);
else
    console.log(`${target} isn't on the array`);

File: C++/LinearSearch.cpp

#include <iostream>

int main()
{
    int arr[] = {1, 3, 2, 5, 6, 9, 0, 8, 7, 4};
    size_t arr_length = sizeof(arr) / sizeof(arr[0]);

    int target = -1;
    int found = -1;
    for (int i = 0; i < arr_length; i++)
    {
        if (arr[i] == target)
        {
            found = i;
            break;
        }
    }

    for (int i = 0; i < arr_length; i++)
    {
        std::cout << arr[i] << " ";
    }

    std::cout << std::endl;

    if (found != -1)
        std::cout << target << " is in the index " << found;
    else
        std::cout << target << " isn't on the array";

    return 0;
}